package/CHANGELOG

```diff
@@ -36,6 +36,8 @@ Enhancements
   * Added computation of Mean Squared Displacements (#2438, PR #2619)
   * Improved performances when parsing TPR files (PR #2804)
   * Added converter between Cartesian and Bond-Angle-Torsion coordinates (PR #2668)
+  * Added a simple version of the RDKitConverter that handles uncharged 
+    molecules with bond orders/bond types assigned (#2468, PR #2775)
```

Comment:
I think the convention we've taken is "Issue #<number>", it's pedantic but probably worth adding for consistency.